mlflow/pyfunc/__init__.py

```diff
@@ -1018,6 +1018,15 @@ def get_raw_model(self):
         raise NotImplementedError("`get_raw_model` is not implemented by the underlying model")
 
 
+class DummyPythonModel(PythonModel):
```

Comment:
Will we remove this in the future? It's a bit weird to put this here since it'll be packed

Reply:
No, it won't be removed. Put this here for convenience.

Reply:
Moved this in tests/pyfunc/test_model_export_with_class_and_artifacts.py.